packages/broker/src/ConfigWizard.ts

```diff
@@ -0,0 +1,195 @@
+            "ws://127.0.0.1:30303"
+        ],
+        location: {
+            latitude: 60.19,
+            longitude: 24.95,
+            country: "Finland",
+            city: "Helsinki"
+        } // TODO
+    },
+    reporting: {
+        intervalInSeconds: 0,
+        streamr: null,
+        perNodeMetrics: null
+    },
+    streamrUrl: 'http://127.0.0.1', // TODO
+    streamrAddress: '0xFCAd0B19bB29D4674531d6f115237E16AfCE377c', // TODO
+    storageNodeConfig: {
+        registry: {
+            contractAddress: '0xbAA81A0179015bE47Ad439566374F2Bae098686F', // TODO
+            jsonRpcProvider: 'http://10.200.10.1:8546' // TODO
+        }
+    },
+    httpServer: {
+        privateKeyFileName: null,
+        certFileName: null,
+        port: TODO_VALUES.number//'{HTTP_SERVER_PORT}',
+    },
+    apiAuthentication: null,
+    plugins: {}
+}
+
+export class ConfigWizard{
+    config: Config
+    // required to mock results on tests
+    inquirer: inquirer.Inquirer = inquirer
+
+    defaultWebsocketPort = 7171 
+    defaultMqttPort = 7272 
+    defaultHttpPort = 7373
+
+    constructor(){
+        this.config = DefaultConfig
+    }
+
+    async inquirerSinglePrompt(prompt: inquirer.Question | inquirer.ListQuestion | inquirer.CheckboxQuestion) {
+        const answers = await this.inquirer.prompt([prompt])
+        return answers[prompt.name!]
+    }
+
+    async generateOrImportPrivateKey(): Promise<string>{
+        const generateOrImport = await this.inquirerSinglePrompt({
+            type: 'list',
+            name:'generateOrImportEthereumPrivateKey',
+            message: 'Do you want to generate a new Ethereum private key or import an existing one?',
+            choices: ['Generate', 'Import'],
+            filter: (choice: string) => {
+                return choice.toLowerCase()
+            }
+        })
+
+        if (generateOrImport === 'generate'){
+            this.config.ethereumPrivateKey = Wallet.createRandom().privateKey
+        } else if (generateOrImport === 'import'){
+            const privateKey = await this.inquirerSinglePrompt({
+                type: 'input',
+                name: 'privateKey',
+                message: "'Please provide the private key to import'",
+            })
+
+            try {
+                const wallet = new Wallet(privateKey)
+                this.config.ethereumPrivateKey = wallet.privateKey
+            } catch (privateKeyError){
+                throw new Error(`Invalid privateKey provided for import: ${privateKey}`)
+            }
+
+        } else {
+            throw new Error(`Invalid option ${generateOrImport} provided`)
+        }
+
+        return this.config.ethereumPrivateKey
+    }
+
+    async promptNumberWithDefault(prompt: inquirer.InputQuestion, defaultValue: number){
+        const valueString = await this.inquirerSinglePrompt(prompt)
+        let value = parseInt(valueString)
+        if (valueString === ''){
+            value = defaultValue
+        }
```

Comment:
No need to run parseInt if it is empty
```suggestion
        const value = valueString === '' ? defaultValue : parseInt(valueString)
```